sdk/spring/azure-spring-boot-test-aad/src/test/java/com/azure/test/aad/selenium/oauth2client/issuedat/AADOauth2AuthorizedClientCachedIT.java

```diff
@@ -62,7 +61,6 @@ public void destroy() {
         aadSeleniumITHelper.destroy();
     }
 
-    @EnableGlobalMethodSecurity(securedEnabled = true, prePostEnabled = true)
```

Comment:
any reason you removed this from those tests?

Reply:
I added the annotation into the default adapter, so other using this annotation can be removed if no customized adapter. https://github.com/moarychan/azure-sdk-for-java/blob/3e35d9a5c3de255044c95ebade405f95e762dcfc/sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/aad/webapp/AADWebAppConfiguration.java#L51